packages/kubetail/v0.5.6+1/package.yaml

```
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: "kubetail"
+shortDescription: "Kubetail is a web-based, real-time log viewer for Kubernetes clusters"
+iconUrl: "https://avatars.githubusercontent.com/u/141319696"
+defaultNamespace: "kubetail"
+helm:
+  repositoryUrl: "https://kubetail-org.github.io/helm-charts/"
+  chartName: "kubetail"
+  chartVersion: "0.5.6"
+entrypoints:
+  - serviceName: kubetail
+    port: 4000
```

When trying to open this entrypoint, I get an error:
```
Could not open package kubetail: could not open entrypoint ui: service kubetail-kubetail has no port 4000
```

It looks like the service only has port 80

```suggestion
    port: 80
    localPort: 4000
```

---

Thanks true! 
Should be fixed now